source/css/scss/app/pages/invite.scss

```diff
@@ -38,6 +38,66 @@ div.invitedIntro {
 	}
 }
 
+div.invite-about {
+	overflow: hidden;
+	background-color: $green;
+	padding-top: 30px;
```

Comment:
@foochris I think we need to get out of the px biz...rem-calc() all of these please.